setuptools/tests/test_build_py.py

```diff
@@ -25,6 +25,29 @@ def test_directories_in_package_data_glob(tmpdir_cwd):
     dist.run_commands()
 
 
+def test_recursive_in_package_data_glob(tmpdir_cwd):
+    """
+    Files matching recursive globs (**) in package_data should
+    be included in the package data.
+
+    #1806
+    """
+    dist = Distribution(dict(
+        script_name='setup.py',
+        script_args=['build_py'],
+        packages=[''],
+        package_data={'': ['path/**/data']},
+    ))
+    os.makedirs('path/subpath/subsubpath')
+    open('path/subpath/subsubpath/data', 'w').close()
+
+    dist.parse_command_line()
+    dist.run_commands()
+
+    assert stat.S_IREG(os.stat('build/lib/path/subpath/subsubpath/data').st_mode), \
```

Comment:
Would the following be the case?
```suggestion
    assert stat.S_ISREG(os.stat('build/lib/path/subpath/subsubpath/data').st_mode), \
```

I could not find a `S_IREG` function in the docs...

Reply:
Consider me embarassed; I switched from `... & S_IREG` to the function call instead and didn't run tox again. I'll change this shortly.